integration-tests/kafka/pom.xml

```diff
@@ -221,7 +227,6 @@
                                     <goal>verify</goal>
                                 </goals>
                                 <configuration>
-                                    <skip>false</skip>
```

Comment:
Shouldn't it be `true` instead?

Reply:
`true` is set above - https://github.com/quarkusio/quarkus/blob/master/integration-tests/kafka/pom.xml#L169

will double check, looked at dynamodb setup and it is using the same approach, no explicit skip element in native profile

Reply:
This is not needed

```
mvn verify -Dnative
...
[quarkus-integration-test-kafka-999-SNAPSHOT-runner:53801]        write:   2,027.51 ms
[quarkus-integration-test-kafka-999-SNAPSHOT-runner:53801]      [total]: 238,161.34 ms
[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 242518ms
[INFO]
[INFO] --- maven-failsafe-plugin:2.22.1:integration-test (default) @ quarkus-integration-test-kafka ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-failsafe-plugin:2.22.1:verify (default) @ quarkus-integration-test-kafka ---
[INFO] Tests are skipped.
...
```

Reply:
Ah, yes, missed the one above.